reference/3.0/Microsoft.PowerShell.Core/Update-Help.md

```diff
@@ -75,127 +84,139 @@ This cmdlet was introduced in Windows PowerShell 3.0.
+language that is set in Windows.
 
 ```powershell
 Update-Help
 ```
 
 ### Example 2: Update help for specified modules
 
-This command updates help only for the **ServerManager** module and for modules that have names that begin with "Microsoft.PowerShell".
-
-Because these modules are in the `$pshome\Modules` directory, to run this command, start PowerShell with the "Run as administrator" option.
+This command updates help only for the **ServerManager** module and for modules that have names that
+begin with **Microsoft.PowerShell**.
 
 ```powershell
 Update-Help -Module ServerManager, Microsoft.PowerShell*
 ```
 
-### Example 3: Update help in different  languages
+### Example 3: Update help in different languages
 
 This command updates the Japanese and English help files for all modules.
 
-If a module currently does not provide help files for the specified UI culture, the error message lists the UI cultures that the module supports.
-In this example, the error message indicates that the **ServerManager** module currently provides help files only in "en-US".
+If a module currently doesn't provide help files for the specified UI culture, the error message
+lists the UI cultures that the module supports. In this example, the error message indicates that
+the ServerManager module currently provides help files only in **en-US**.
 
 ```powershell
 Update-Help -UICulture ja-JP, en-US
 ```
 
-```output
+```Output
 Update-Help : Failed to update Help for the module(s) 'ServerManager' with UI culture(s) {ja-JP} :
 The specified culture is not supported: ja-JP. Specify a culture from the following list: {en-US}.
 ```
 
 ### Example 4: Update help automatically
 
-This command creates a scheduled job that updates help for all modules on the computer every day at 3:00 in the morning.
-
-The command uses the `Register-ScheduledJob` cmdlet to create a scheduled job that runs an `Update-Help` command.
-The command uses the **Credential** parameter to run the `Update-Help` cmdlet with the credentials of a member of the "Administrators" group on the computer.
-The value of the **Trigger** parameter is a `New-JobTrigger` command that creates a job trigger that starts the job every day at "3:00 AM".
-
-To run the `Register-ScheduledJob` command, start PowerShell with the "Run as administrator" option.
-When you run the command, PowerShell prompts you for the password of the user specified in the value of the **Credential** parameter.
-The credentials are stored with the scheduled job; you are not prompted when the job runs.
-
-You can use the `Get-ScheduledJob` cmdlet to view the scheduled job, use the `Set-ScheduledJob` cmdlet to change it, and use the `Unregister-ScheduledJob` cmdlet to delete it.
-You can also view and manage the scheduled job in Task Scheduler in the following path: "Task Scheduler Library\Microsoft\Windows\PowerShell\ScheduledJobs".
+This command creates a scheduled job that updates help for all modules every day at 3:00 AM.
```

Comment:
Style change here - Rather than saying "This command" we should say "This example". Because there could be multiple commands in the example.

Reply:
Resolved in updated doc.